.ci/install_kata.sh

```diff
@@ -28,6 +28,10 @@ if [ "${TEE_TYPE:-}" == "tdx" ]; then
 	KATA_BUILD_QEMU_TYPE="${KATA_BUILD_QEMU_TYPE:-tdx}"
 fi
 
+if [ "${TEE_TYPE:-}" == "sev" ]; then
+	KATA_BUILD_KERNEL_TYPE="${KATA_BUILD_KERNEL_TYPE:-sev}"
```

Comment:
Wouldn't you want this to be `KATA_BUILD_KERNEL_TYPE=sev`  instead of using the previous value if set?